### PR TITLE
feat: Update anchor data model

### LINKED ISSUE #581: Update anchor data model
Sample:

```
{
    "@context": [
        "https://www.w3.org/ns/activitystreams",
        "https://w3id.org/activityanchors/v1"
    ],
    "type": "AnchorEvent",
    "attributedTo": "ipns://k51qzi5uqu5dl3ua2aal8vdw82j4i8s112p495j1spfkd2blqygghwccsw1z0p",
    "published": "2021-01-27T09:30:00Z",
    "previous": [{
        "id": "urn:multihash:u2ouBBsyLDedeYciUmaihjWUmWhA3LVkruPZwSk7EeHWWqUZF",
        "url": "ipfs://bafkreicdkwsgwgotjdoc6v6ai34o6y6ukohlxe3aadz4t3uvjitumdoymu",
        "type": "Link"
    }],
    "attachment": [{
        "type": "AnchorIndex",
        "generator": "https://w3id.org/orb#v1",
        "url": [
            {
                "href": "ipfs://bafkreihwsnuregceqh263vgdathcprnbvatyat6h6mu7ipjhhodcdbyhoy",
                "type": "Link",
                "rel": "self"
            },
            {
                "href": "https://example.com/cas/bafkreihwsnuregceqh263vgdathcprnbvatyat6h6mu7ipjhhodcdbyhoy",
                "type": "Link",
                "rel": "alternate"
            }
        ],
        "resources": [
            "urn:multihash:uEiDahaOGH-liLLdDtTxEAdc8i-cfCz-WUcQdRJheMVNn3A",
            {
                "id": "urn:multihash:uEiA329wd6Aj36YRmp7NGkeB5ADnVt8ARdMZMPzfXsjwTJA",
                "previousAnchor": "urn:multihash:u2ouBBsyLDedeYciUmaihjWUmWhA3LVkruPZwSk7EeHWWqUZF",
                "type": "AnchorResource"
            }
        ]
    }]
}
```

### PR DESCRIPTION
**Pull request by sandrask**
Update anchor data model:
- add/load new context(s)
- create anchor activity model
- transform from current payload to anchor activity
- update unit tests

Closes #581

Signed-off-by: Sandra Vrtikapa <sandra.vrtikapa@securekey.com>

### CODE REVIEW

**Review by fqutishat — APPROVED** [review]

File: pkg/anchor/activity/activity.go [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+
+	linkType = "Link"
+
+	idKey             = "id"
+	previousAnchorKey = "previousAnchor"
+	resourceTypeKey   = "type"
+)
+
+// Activity defines anchor activity.
+type Activity struct {
```
Comment (bstasyszyn) [inline pkg/anchor/activity/activity.go:35]: This isn't really an activity, is it? This object will be embedded within the 'Offer' activity.

File: internal/pkg/ldcontext/testdata/activity-streams.json [hunk 2]
```diff
@@ -0,0 +1,382 @@
+{
```
[the remark this answers is not in the document]
Comment (sandrask) [reply internal/pkg/ldcontext/testdata/activity-streams.json:1]: This is how the current contexts are loaded.

File: pkg/anchor/activity/activity.go (hunk 1 above)
Comment (sandrask) [reply pkg/anchor/activity/activity.go:35]: It is called anchor activity so I put it in activity package under anchor package. Any suggestions for naming?

File: internal/pkg/ldcontext/testdata/activity-streams.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (sandrask) [reply internal/pkg/ldcontext/testdata/activity-streams.json:1]: I can create new issue to re-visit how it is done.

[the remark this answers is not in the document]
Comment (bstasyszyn) [reply internal/pkg/ldcontext/testdata/activity-streams.json:1]: Ok, we should at one point rename this - I thought it was just used for unit tests.

File: pkg/anchor/activity/activity.go (hunk 1 above)
Comment (bstasyszyn) [reply pkg/anchor/activity/activity.go:35]: After reading the new spec again, I see that it is called an anchor activity. @troyronda I'm not sure that this is an activity, is it? I believe activities are like actions.

**Review by bstasyszyn — APPROVED** [review]

File: internal/pkg/ldcontext/testdata/activity-streams.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (sandrask) [reply internal/pkg/ldcontext/testdata/activity-streams.json:1]: Created #610

File: pkg/anchor/activity/activity.go (hunk 1 above)
Comment (sandrask) [reply pkg/anchor/activity/activity.go:35]: Created #611

**Pull request merged**